src/org.spec.ts

```diff
@@ -507,19 +508,29 @@ describe('org', () => {
         memberAccessType: MemberAccessType.OPEN,
       });
 
-      const methodResponse = await deleteOrg(ORG_NAME, {
+      const result = await deleteOrg(ORG_NAME, {
         dbConnection: connection,
         logger: mockLogging.logger,
       });
 
-      requireSuccessfulResult(methodResponse);
+      requireSuccessfulResult(result);
       const dbResult = await orgModel.exists({
         name: NON_ASCII_ORG_NAME,
       });
       expect(dbResult).not.toBeNull();
     });
 
-    test('Record Find errors should be propagated', async () => {
+    test('Non existing name should not produce logs', async () => {
```

Comment:
Keep in mind that the convention is not to create a test just to check logs, but to check the log in the appropriate test.